.github/workflows/ci.yaml

```diff
@@ -282,7 +282,7 @@ jobs:
       run: |
         mvn install -B -ntp -DskipTests -Dclirr.skip -Dcheckstyle.skip
     - name: Validate gapic-generator-java-bom
-      uses: googleapis/java-cloud-bom/tests/validate-bom@v26.13.0
+      uses: googleapis/java-cloud-bom/tests/validate-bom
```

Comment:
In the document https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_iduses I don't see the "ref" after "@" is optional.